.nightfalldlp/config.json

```diff
@@ -6,5 +6,7 @@
     "tokenExclusionList": [
         "4242-4242-4242-4242",
         "4916-6734-7572-5015"
-    ]
+    ],
+    "fileInclusionList": [".*"],
+    "fileExclusionList": ["exclude\\.txt"]
```

Comment:
Im not sure I understand why there is `exclude\\` in the front of this

Reply:
ya i made a text file named `exclude.txt`, the `\\` is to treat the `.` as the literal dot